### PR TITLE
adding a new stage in api manager pipeline for logging

### PR DESCRIPTION
**Pull request by bilalahmad99**
@edx/devops @efagin

### CODE REVIEW

File: edxpipelines/pipelines/api_deploy.py [hunk 1]
```diff
@@ -158,8 +160,40 @@ def install_pipeline(save_config_locally, dry_run, variable_files, cmd_line_vars
...
+        }
+    )
+
+    log_gateway_job.add_task(FetchArtifactTask(**api_manager_artifact_params))
 
+    log_gateway_job.add_task(
+        ExecTask(
+            [
+                '/bin/bash', '-c',
+                'PYTHONPATH=python-libs python scripts/aws/monitor.py --api-base-domain ${API_BASE} --splunk-host ${splunk-host} --splunk-token ${splunk-token} --acct-id ${acct-id} --kms-key ${kms-key} --subnet-list ${subnet-list} --sg-list ${sg-list} --environment ${environment} --deployment ${deployment}'
```
Comment (efagin) [inline edxpipelines/pipelines/api_deploy.py]: @bilalahmad99 where are you passing in `API_BASE`? The other environment vars have all been declared above, but not this one.

Comment (efagin) [inline edxpipelines/pipelines/api_deploy.py]: Sorry - please disregard. I realized that that variable was declared further up in this file (all other vars are more local to this job, which makes sense). This looks good to me!

**Comment by efagin** [comment]
👍 - please merge after the vars PR has merged.

**Comment by feanil** [comment]
:+1:

**Comment by efagin** [comment]
@bilalahmad99 did you catch this dash bug in production? or during your testing? (I'm surprised this didn't come up during your testing and am wondering what is different between environments.)

**Comment by bilalahmad99** [comment]
@efagin no i caught it in during my testing with the help of Feanil. I havent incorporated my changes to production, but I am about to.

**Pull request merged**